libs/core/tests/unit_tests/prompts/test_chat.py

```diff
@@ -863,3 +875,32 @@ async def test_chat_tmpl_serdes(snapshot: SnapshotAssertion) -> None:
     )
     assert dumpd(template) == snapshot()
     assert load(dumpd(template)) == template
+
+
+def test_chat_content_block() -> None:
```

Comment:
should we write some serialization snapshot tests and run them before and after this pr?

Reply:
already done (see test above)